docs/linters.md

```diff
@@ -0,0 +1,210 @@
+# Linters
```

Comment:
I'm gonna add this maybe lower in the document than you were envisioning, because I think a larger percentage of people will need to know specifics about how to run our linters than understanding the overall philosophy, which is really fairly common to any project that does heavy linting.